packages/components/src/dropdown-menu-v2/styles.ts

```diff
@@ -304,6 +304,15 @@ export const DropdownMenuGroup = styled( Ariakit.MenuGroup )`
 	display: contents;
 `;
 
+export const DropdownMenuGroupLabel = styled( Ariakit.MenuGroupLabel )`
+	/* Occupy the width of all grid columns (ie. full width) */
+	grid-column: 1 / -1;
+
+	padding-block-start: ${ space( 3 ) };
+	padding-block-end: ${ space( 2 ) };
```

Comment:
I guess we're intentionally not using `ITEM_PADDING_BLOCK` / `ITEM_PADDING_INLINE` for those?

Reply:
We are using `ITEM_PADDING_INLINE` to align the label horizontally, but the block padding is custom